src/main/webapp/manage-availability.js

```diff
@@ -91,6 +97,28 @@ function createTimeSlotBoxManage(timeslot, tutorID) {
     return timeslotElement;
 }
 
+function addTimeSlot() {
+    var queryString = new Array();
+    window.onload = readTutorID(queryString, window);
+    const userID = queryString["userID"];
+
+    const params = new URLSearchParams();
+
+    params.append('startHour', document.getElementById('startHour').value);
+    params.append('startMinute', document.getElementById('startMinute').value);
+    params.append('endHour', document.getElementById('endHour').value);
+    params.append('endMinute', document.getElementById('endMinute').value);
+    params.append('day', document.getElementById('day').value);
+    params.append('month', document.getElementById('month').value);
+    params.append('year', document.getElementById('year').value);
+    params.append('tutorID', userID);
+
+    fetch('/add-availability', {method: 'POST', body: params}).then(() => {
+        window.location.href = "/manage-availability.html?userID=" + userID;
+    });
+
+}
+
```

Comment:
I was a using a post request on the form to add the availability but this looks like a good way to do it too. Was it mostly for readability or was there another reason for choosing this approach?

Reply:
I couldn't send the tutorID when we were using the post request on the form, but I agree that way was definitely easier.